From 042795c3d5a9048c5dc1b3ddeb63f17c22f8374b Mon Sep 17 00:00:00 2001
From: Matthias Brugger <mbrugger@suse.com>
Date: Fri, 17 Jan 2020 20:59:02 +0100
Subject: [PATCH] Kconfig: add btrfs to distro boot

Some distributions use btrfs as the default file system.
Enable btrfs support by default when using distro boot for all
architectures but riscv, as it breaks compilation due to size problems.

Signed-off-by: Matthias Brugger <mbrugger@suse.com>
---
 Kconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Kconfig b/Kconfig
index 86f0a39bb0..fea92305a3 100644
--- a/Kconfig
+++ b/Kconfig
@@ -143,6 +143,7 @@ config DISTRO_DEFAULTS
 	select HUSH_PARSER
 	select SUPPORT_RAW_INITRD
 	select SYS_LONGHELP
+	imply CMD_BTRFS if !RISCV && !MIPS
 	imply CMD_MII if NET
 	imply USB_STORAGE
 	imply USE_BOOTCOMMAND
